paasta_tools/kubernetes/bin/paasta_secrets_sync.py

```diff
@@ -99,7 +101,13 @@ def parse_args() -> argparse.Namespace:
     )
     parser.add_argument(
         "--secret-type",
-        choices=["all", "paasta-secret", "boto-key", "crypto-key"],
+        choices=[
+            "all",
+            "paasta-secret",
+            "boto-key",
+            "crypto-key",
+            "datastore-credentials",
+        ],
         default="all",
         type=str,
         help="Define which type of secret to add/update. Default is 'all'",
```

Comment:
++ fair point! will add that it. what I've gleaned from the conversations with @nemacysts and @mattmb is there may be a move away from using 'all' and separating the CRONs entirely! not sure if I paraphrased correctly though

Reply:
That's quite true, since these are now split out, we should be able to remove the 'all' option entirely Soon:tm:, though don't know exactly when that will happen. Feel free to ignore me if we think we're near that point already.

Reply:
I'm thinking of making a PR next week once the split cronjobs have been live for a couple days - but we've been using N different cronjobs since yesterday with no issue so far :)

Reply:
In that case, resolving this comment 👍